backend/test/routes/test_allocation.py

```diff
@@ -0,0 +1,40 @@
+import json
+import unittest
+
+from models.abc import db
+from repositories import TickerCovarianceRepository, TickerRepository
+from server import server
+
+
+class TestTicker(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
```

Comment:
are these db methods usually camelCased?

Reply:
This is from the default project so i assume its in the correct style